docs/rfcs/20210223_external_connectivity_in_k8s.md

```diff
@@ -0,0 +1,230 @@
+- Feature Name: Rafał Korepta
+- Status: in-progress
+- Start Date: 2021-02-23
+- Authors: RafalKorepta
+- Issue: (one or more # from the issue tracker)
+
+# Executive Summary
+
+Users would like to managed Redpanda clusters in kubernetes environment,
+but the clients could live outside of the kubernetes network boundary.
+To give users out of the box experience Redpanda kubernetes operator
+should pave the way for the networking stack to expose each Redpanda
+node in k8s environment.
+
+## What is being proposed
+
+The most performant solution is to expose each process to the outside,
+by specifying
+[host port in the container ports specification](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#containerport-v1-core)
+for each Redpanda node. Each instance will
```

Comment:
dummy question: why do we need dns? that could still work with nodeip+nodeport, right?

Reply:
You are totally right. I have not good answer to that. I will remove external dns dependency